crates/stages/src/stages/execution.rs

```diff
@@ -73,6 +75,10 @@ pub struct ExecutionStage<EF: ExecutorFactory> {
     external_clean_threshold: u64,
     /// Pruning configuration.
     prune_modes: PruneModes,
+    /// Handle to communicate with ExEx manager.
+    ///
+    /// TODO(onbjerg): Unbox this when we can remove the temporary trait
+    exex_manager_handle: Option<Box<dyn TempManagerHandle>>,
```

Comment:
I went for boxing here instead of a generic because the generic would be a lot more clean up